reactive/kubernetes_master.py

```diff
@@ -2162,6 +2187,9 @@ def configure_controller_manager():
     controller_opts['tls-cert-file'] = str(server_crt_path)
     controller_opts['tls-private-key-file'] = str(server_key_path)
     controller_opts['cluster-name'] = leader_get('cluster_tag')
+    controller_opts['terminated-pod-gc-threshold'] = '12500'
+    controller_opts['profiling'] = 'false'
+    controller_opts['feature-gates'] = 'RotateKubeletServerCertificate=true'
```

Comment:
Not sure what the impact of enabling `RotateKubeletServerCertificate` will be for us, given that we pass our own certificate into Kubelet rather than allowing it to generate its own certificate through the k8s Certificates API.

Hopefully this is a no-op, but we should keep our eye out for cert issues relating to Kubelet. In particular, we should make sure the `kubectl logs` and `kubectl exec` commands still work.

Reply:
I'm not sure either.  Both `logs` and `exec` work though.  Is it a worry they'll work until rotated?